amundsen_application/static/js/ducks/notification/api/v0.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+import { NotificationType, SendNotificationOptions } from 'interfaces/Notifications'
```

Comment:
Checkout `interfaces/index.ts`. You can export the `Notifications` module from there and just be able to use `... from 'interfaces'`.